Editor/UI/Wizard/FirstTime/TrainingSceneSetupPage.cs

```diff
@@ -24,58 +24,53 @@ public TrainingSceneSetupPage() : base("Step 1: Sample Training")
         /// <inheritdoc />
         public override void Draw(Rect window)
         {
-            string spaceAfterRadioButton = "  ";
-
             GUILayout.BeginArea(window);
 
             GUILayout.Label("Load a sample training", CreatorEditorStyles.Title);
-            GUILayout.Space(verticalSpace);
-                GUILayout.BeginHorizontal();
-                GUILayout.Space(horizontalSpace);
-                    GUILayout.BeginVertical();
-
-                    createNewTraining = GUILayout.Toggle(createNewTraining, spaceAfterRadioButton + "Load sample VR training (recommended)", EditorStyles.radioButton);
-                    createNewTraining = GUILayout.Toggle(!createNewTraining, spaceAfterRadioButton + "Start from scratch with an empty VR training", EditorStyles.radioButton);
-
-                    GUILayout.Space(verticalSpace);
-
-                    GUILayout.EndVertical();
-                GUILayout.EndHorizontal();
 
-                GUILayout.BeginHorizontal();
-                GUILayout.Space(horizontalSpace * 2);
-                    GUILayout.BeginVertical();
+            // Use the next two lines and remove the "loadSample = false" line as soon as loading samples is supported
+            // if (GUILayout.Toggle(loadSample, "Load sample VR training (recommended)", CreatorEditorStyles.Toggle)) loadSample = true;
+            // if (GUILayout.Toggle(!loadSample, "Start from scratch with an empty VR training", CreatorEditorStyles.Toggle)) loadSample = false;
+            loadSample = false;
 
-                    GUILayout.Label("Name of your VR Training:");
-                    courseName = GUILayout.TextField(courseName, 30, GUILayout.Width(window.width * 0.7f));
-
-                    string subText;
-
-                    if (SceneExists(courseName) && useCurrentScene)
-                    {
-                        subText = "Scene already exists.";
-                        CanProceed = false;
-                    }
-                    else if (CourseAssetUtils.DoesCourseAssetExist(courseName))
+            if (loadSample == false)
+            {
+                RectOffset margin = CreatorEditorStyles.Paragraph.margin;
+                margin.top = CreatorEditorStyles.BaseMargin + CreatorEditorStyles.Indent;
+                GUILayout.Label("Name of your VR Training:",
+                    CreatorEditorStyles.ApplyMargin(CreatorEditorStyles.Paragraph, margin));
+                courseName = GUILayout.TextField(courseName, 30,
```

Comment:
Where is this 30 character limit from?
No Magic number please
Why does the text field size actually give you the impression that you are allowed to fit 150 character in there but only 30 are allowed (which are btw 3 more then the example text)

Reply:
It is a magic number because it prevents people from using 8329472398047 characters which will most likely not work 🤷 
I would set a limit and I also do think a magic number is fine here, why not? We can increase it but I am totally against unlimited

Reply:
https://wiki.c2.com/?MagicNumber

Reply:
And if we keep using a character limit of 30 i would still like to see the text field reflecting this in size. Actually looks broken when I cant type anymore after using only 1/3 of the text field.